src/main/java/programminglife/utility/Alerts.java

```diff
@@ -25,32 +24,26 @@ private Alerts() { }
     /**
      * Alert method used if the alert is an error.
      * @param message String containing the message to be given to the user.
-     * @return Alert error.
      */
-    @NotNull
-    public static Alert error(String message) {
-        return new Alert(Alert.AlertType.ERROR, message, ButtonType.CLOSE);
+    public static void error(String message) {
+        Platform.runLater(() -> new Alert(Alert.AlertType.ERROR, message, ButtonType.CLOSE).show());
     }
 
     /**
      * Alert method used if the alert is a warning.
      * @param message String containing the message to be given to the user.
-     * @return Alert warning.
      */
-    @NotNull
-    public static Alert warning(String message) {
-        return new Alert(Alert.AlertType.WARNING, message, ButtonType.CLOSE);
+    public static void warning(String message) {
+        Platform.runLater(() -> new Alert(Alert.AlertType.WARNING, message, ButtonType.CLOSE));
     }
 
 
     /**
      * Alert method used if the alert is informative.
      * @param message String containing the message to be given to the user.
-     * @return Alert info.
      */
-    @NotNull
-    public static Alert info(String message) {
-        return new Alert(Alert.AlertType.INFORMATION, message, ButtonType.CLOSE);
+    public static void info(String message) {
+        Platform.runLater(() -> new Alert(Alert.AlertType.INFORMATION, message, ButtonType.CLOSE));
     }
```

Comment:
I think that either there should be `.show()` for `warning()` and `info()`, or it shouldn't be there in `error()`